code/src/components/Age.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+const Age = ({ ageGroup, setAgeGroup, nextSlide, backSlide }) => {
+  const ageSelectors = ['0-20', '21-40', '41-60', '60+']
+
+    return (
+     <section>
+      <div className='form-container'>
+        <h2>Select your age</h2>
+        {ageSelectors.map(group => (
+        <label key={group} htmlFor='age'>
+        <input className='radio-input'
+          id='age'
+          type='radio'
+          value={group}
+          onChange={(event) => {setAgeGroup(event.target.value)}}
+          checked={ageGroup === group}
+        />
+        {group}
+        </label>
+        ))}
+        <div className='button-container'>
```

Comment:
The idea of using 'back' and 'next' buttons completes the user experience. If the user wants to change the previous answer there is a possibilty to do so. Great function to your survey!